src/Database/Connections/MariaDbConnection.php

```diff
@@ -0,0 +1,87 @@
+<?php namespace Winter\Storm\Database\Connections;
+
+use PDO;
+use Illuminate\Database\Schema\MariaDbBuilder;
+use Illuminate\Database\Query\Processors\MariaDbProcessor;
+
+use Winter\Storm\Database\PDO\MySqlDriver;
+use Winter\Storm\Database\Schema\Grammars\MariaDbGrammar as SchemaGrammar;
+use Winter\Storm\Database\Query\Grammars\MariaDbGrammar as QueryGrammar;
+
+/**
+ * @phpstan-property \Illuminate\Database\Schema\Grammars\Grammar|null $schemaGrammar
+ */
+class MariaDbConnection extends \Illuminate\Database\MariaDbConnection
+{
```

Comment:
Should this class use the Connection trait as well?

Reply:
Yup